### PR TITLE
brad publishes CIAM article

### PR DESCRIPTION
**Pull request by bradmccarty**
Publishing Mark R's "What is CIAM" article to /articles/CIAM

### CODE REVIEW

File: astro/src/content/articles/ciam/what-is-ciam.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+description: "CIAM solutions help organizations to securely authenticate and authorize customers, protect customer data, and provide a seamless customer experienc"
+section: CIAM
+author: Mark Robustelli 
+icon: /img/icons/what-is-ciam.svg
+---
+Customer Identity and Access Management (CIAM) is a set of processes and technologies that organizations use to manage the identities and access of their customers. CIAM solutions help organizations to securely authenticate and authorize customers, protect customer data, and provide a seamless customer experience.
+
+
+## 
+**What is CIAM?**
```
Comment (mooreds) [inline astro/src/content/articles/ciam/what-is-ciam.md:12]: This should be on previous line and doesn't need `**`.

File: astro/src/content/articles/ciam/what-is-ciam.md [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+
+Customer Identity and Access Management (CIAM) is a set of technologies and processes that organizations use to manage the identities and access of their customers. CIAM solutions help organizations to:
+
+
```
Comment (mooreds) [inline astro/src/content/articles/ciam/what-is-ciam.md:16]:
```suggestion
```

File: astro/src/content/articles/ciam/what-is-ciam.md [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+icon: /img/icons/what-is-ciam.svg
+---
+Customer Identity and Access Management (CIAM) is a set of processes and technologies that organizations use to manage the identities and access of their customers. CIAM solutions help organizations to securely authenticate and authorize customers, protect customer data, and provide a seamless customer experience.
+
+
+## 
+**What is CIAM?**
+
+Customer Identity and Access Management (CIAM) is a set of technologies and processes that organizations use to manage the identities and access of their customers. CIAM solutions help organizations to:
+
```
Comment (mooreds) [inline astro/src/content/articles/ciam/what-is-ciam.md:15]:
```suggestion
```

No need for 3 empty lines.

File: astro/src/content/articles/ciam/what-is-ciam.md [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+
+
+* Secure customer identities: CIAM solutions can help to protect customer identities from unauthorized access, use, or disclosure. This can be done through the use of strong authentication methods, such as multi-factor authentication (MFA), and by implementing other security measures, such as password policies and session management.
+* Provide a seamless customer experience: CIAM solutions can help to improve the customer experience by making it easier for customers to sign up, log in, and access services. This can be done by providing a single sign-on (SSO) solution, by allowing customers to register and log in using social media accounts, and by providing self-service tools for managing customer profiles.
+* Gather customer data: CIAM solutions can help organizations to gather customer data, such as contact information, preferences, and purchase history. This data can be used to improve the customer experience, to target marketing campaigns, and to make better business decisions.
+
+### 
+**Benefits to using CIAM solutions**
```
Comment (mooreds) [inline astro/src/content/articles/ciam/what-is-ciam.md:23]: Same issue -- put it on the same line and remove the `**`.

File: astro/src/content/articles/ciam/what-is-ciam.md [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+
+* Improved security: CIAM solutions can help to protect customer identities and data from unauthorized access, use, or disclosure. This can help to reduce the risk of fraud, data breaches, and other security incidents.
+* Enhanced customer experience: CIAM solutions can help to improve the customer experience by making it easier for customers to sign up, log in, and access services. This can lead to increased customer satisfaction and loyalty.
+* Increased business agility: CIAM solutions can help organizations to be more agile by making it easier to onboard new customers, launch new products and services, and respond to changing customer demands.
+* Reduced costs: CIAM solutions can help to reduce costs by streamlining customer onboarding and support processes, and by reducing the need for manual intervention.
+
+## 
+**CIAM solutions**
```
Comment (mooreds) [inline astro/src/content/articles/ciam/what-is-ciam.md:31]: Same issue -- put it on the same line and remove the `**`.

File: astro/src/content/articles/ciam/what-is-ciam.md [hunk 6]
```diff
@@ -0,0 +1,222 @@
...
+
+
+
+* FusionAuth is a cloud-native CIAM solution that is designed to be scalable, secure, and easy to use.
+* ForgeRock: ForgeRock is a leading provider of identity and access management solutions. ForgeRock's CIAM solution, Identity Cloud, offers a comprehensive set of features for managing customer identities and access.
+* Ping Identity: Ping Identity is a provider of identity and access management solutions. Ping Identity's CIAM solution, PingOne, offers a secure and user-friendly solution for managing customer identities and access.
+* OneLogin: OneLogin is a provider of identity and access management solutions. OneLogin's CIAM solution, OneLogin, offers a simple and affordable solution for managing customer identities and access.
+
+### 
+**Customer experience**
```
Comment (mooreds) [inline astro/src/content/articles/ciam/what-is-ciam.md:44]: Same issue -- put it on the same line and remove the `**`.

File: astro/src/content/articles/ciam/what-is-ciam.md [hunk 7]
```diff
@@ -0,0 +1,222 @@
...
+
+
```
Comment (mooreds) [inline astro/src/content/articles/ciam/what-is-ciam.md:46]:
```suggestion
```

File: astro/src/content/articles/ciam/what-is-ciam.md [hunk 8]
```diff
@@ -0,0 +1,222 @@
...
+One of the key benefits of CIAM solutions is that they can help to improve the customer experience. CIAM solutions can make it easier for customers to sign up, log in, and access services. This can lead to increased customer satisfaction and loyalty.
+
+There are a number of ways that CIAM solutions can improve the customer experience. For example, CIAM solutions can:
+
```
Comment (mooreds) [inline astro/src/content/articles/ciam/what-is-ciam.md:50]:
```suggestion
```

File: astro/src/content/articles/ciam/what-is-ciam.md [hunk 9]
```diff
@@ -0,0 +1,222 @@
...
+
```
Comment (mooreds) [inline astro/src/content/articles/ciam/what-is-ciam.md:51]:
```suggestion
```

File: astro/src/content/articles/ciam/what-is-ciam.md [hunk 10]
```diff
@@ -0,0 +1,222 @@
...
+
+* Offer a single sign-on (SSO) solution, which allows customers to sign in to multiple applications with a single set of credentials.
+* Allow customers to register and log in using social accounts.
+* Provide self-service tools for managing customer profiles.
+* Personalize the customer experience based on customer preferences.
+
+### 
+**User experience**
```
Comment (mooreds) [inline astro/src/content/articles/ciam/what-is-ciam.md:59]: Same issue -- put it on the same line and remove the `**`.

File: astro/src/content/articles/ciam/what-is-ciam.md [hunk 11]
```diff
@@ -0,0 +1,222 @@
...
+
```
Comment (mooreds) [inline astro/src/content/articles/ciam/what-is-ciam.md:60]:
```suggestion
```

File: astro/src/content/articles/ciam/what-is-ciam.md [hunk 12]
```diff
@@ -0,0 +1,222 @@
...
+
+The user experience (UX) of a CIAM solution is also important. A good UX will make it easy for customers to use the solution and will help to improve the overall customer experience.
+
+There are a number of factors that can affect the UX of a CIAM solution. These factors include:
+
```
Comment (mooreds) [inline astro/src/content/articles/ciam/what-is-ciam.md:65]:
```suggestion
```

File: astro/src/content/articles/ciam/what-is-ciam.md [hunk 13]
```diff
@@ -0,0 +1,222 @@
...
+
```
Comment (mooreds) [inline astro/src/content/articles/ciam/what-is-ciam.md:66]:
```suggestion
```

**Review by mooreds — CHANGES_REQUESTED** [review]
In general, put header text on the same line as `##` and remove the `**`. 

Also, remove extra whitespace as commented.

**Comment by bradmccarty** [comment]
Changes made. I'm still seeing "changes requested" and I don't know how to clear it @mooreds

**Pull request closed without merging**